src/components/ebay-toast/README.md

```diff
@@ -0,0 +1,22 @@
+# ebay-toast
+
+```marko
+<ebay-toast open a11y-close-text="Close Toast">
+  <h1>Hello World</h1>
+</ebay-toast>
+```
+
+## Attributes
+
+Name | Type | Stateful | Required | Description
+--- | --- | --- | --- | ---
+`open` | Boolean | Yes | No | Whether toast is open.
+`focus` | String | No | No | An id for an element which will receive focus when the toast opens (defaults to close button).
+`a11y-close-text` | String | No | Yes | A11y text for close button and mask.
```

Comment:
Remove "and mask". If that was copied over from dialog, it might need removing from there too.